managemon.c

```diff
@@ -575,7 +634,7 @@ static void manage_member(struct mdstat_ent *mdstat,
 				  "sync_action", "recover") == 0)
 			newa->prev_action = recover;
 		dprintf("recovery started on %s\n", a->info.sys_name);
- out:
+
```

Comment:
new line not needed

Reply:
it is needed here, the next instruction is while.